### DISCUSSION TITLE
Wallet not the same as original

### DISCUSSION THREAD

**Discussion by Spacezify** [Chia-Network/chia-blockchain#5054] [Q&A]
Hello! I have a question regarding receive wallet. Few days back my receive wallet is different from my current receive wallet at GUI. Is this supposed to happen? But when i open config.yaml the receive address is the original receive address that i had.

Hope someone can help me with this, a little concern since i checked the current wallet is unsearchable at chiaexplorer.com

**Comment by cccat6** [comment]
You can have multiple receiving addresses for a single wallet. However, it should never change automatically. Did you do any operation on your wallet or did you input the wrong key?

**Reply by Spacezify** [reply]
I did transfer 2XCH into an exchange. I don't remember clicking new receiving address tho. Hmmm

**Reply by cccat6** [reply]
That should be fine. The receiving can be added but not yet able to invalidate.
